pkgs/top-level/all-packages.nix

```diff
@@ -38609,4 +38609,6 @@ with pkgs;
   tubekit = callPackage ../applications/networking/cluster/tubekit/wrapper.nix { };
 
   tubekit-unwrapped = callPackage ../applications/networking/cluster/tubekit { };
+
+  compdb = python3Packages.callPackage ../development/python-modules/compdb { };
```

Comment:
entries in all-packages.nix should be sorted in alphabetical order

Reply:
Roughly sorted, actually. Humans do make mistakes.

Reply:
Moved under `DEVELOPMENT / TOOLS` to a correct alphabetical position.